src/test/java/org/epam/TestngRunner.java

```diff
@@ -0,0 +1,18 @@
+package org.epam;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+import org.testng.annotations.DataProvider;
+
+@CucumberOptions(
+    features = "src/test/resources/features",
+    glue = "org.epam.testng_definitions",
+    plugin = {"pretty", "html:target/cucumber-reports"}
+)
+public class TestngRunner extends AbstractTestNGCucumberTests {
```

Comment:
If you want to run the tests using JUnit as well, you would need the same runner and pre/post conditions for JUnit. If you are not planning to support JUnit, please remove every mention of JUnit in the build.gradle